blocks/callout/callout.css

```diff
@@ -0,0 +1,92 @@
+.callout-container {
+  container-type: inline-size;
+  container-name: callout-container;
+}
+
+.callout {
+  border-radius: var(--spectrum-corner-radius-800);
+  padding: 1.5rem;
+  background: var(--gradient-multitone);
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+
+  .button {
+    align-self: center;
+
+    svg {
+      margin-inline-end: 0.5rem;
+    }
+  }
+  
+  @container callout-container (min-width: 35.5rem) {
+    padding: 2.5rem;
+    flex-direction: row;
+    gap: 2rem;
+  }
+  
+  @container callout-container (min-width: 44rem) {
+    padding: 1.5rem;
+  }
+  
+  @container callout-container (min-width: 49rem) {
+    padding: 3.625rem;
+    justify-content: space-between;
+  }
+}
+
+.callout--link {
```

Comment:
could we name this like `.callout--with-link` so we know it's a modifier and not a subcomponent?